client/src/compontents/login/Login.js

```diff
@@ -1,11 +1,48 @@
-import React,{useState} from 'react';
+import React, { useState } from 'react';
 import styled from 'styled-components';
+import Form from './../../theme/form';
+import Label from './../../theme/label';
+import Input from './../../theme/input';
+import Button from './../../theme/button';
```

Comment:
Hmm could this be compressed into some like `import Form, Label, Input, Button from './../../theme'`

Reply:
good suggestion, will do that!